components/searchbar/index.jsx

```diff
@@ -0,0 +1,12 @@
+import './index.css'
+import React from 'react'
```

Comment:
No, it should not be like this.

Global modules go first, local ones second and stylesheets go the last because it's not even JS.